Editor/CreateNewPrefab.cs

```diff
@@ -5,14 +5,48 @@ public class CreateNewPrefab : EditorWindow
 {
     // -- PRIVATE
 
-    //static string PrefabCreationPath = "Assets/__Content/Prefabs/";
-    static string PrefabCreationPath = "Assets/__Epistory2/Prefabs/Assets/";
+    static string PrefabPathLocalKey = "FC/CommonCode/NewPrefabPath";
+
+    static string PrefabCreationPath = "Assets/_Content/Prefabs/Assets/";
     static string PrefabPrefix = "P_";
     static string PrefabExtension = ".prefab";
 
-
-    static void CreatePrefabs( bool can_be_variant, bool put_in_empty_parent )
+    static void CreatePrefabs(
+        bool can_be_variant,
+        bool put_in_empty_parent,
+        bool it_must_propose_path_selection
+        )
     {
+        string previous_path = EditorPrefs.GetString( PrefabPathLocalKey, PrefabExtension );
+        string destination_path = string.Empty;
+
+        if( it_must_propose_path_selection )
+        {
+            destination_path = EditorUtility.OpenFolderPanel( "Destination Path :", PrefabCreationPath, "" );
+
+            if( string.IsNullOrEmpty( destination_path )
+                || AssetDatabase.IsValidFolder( destination_path )
```

Comment:
shouldn’it be the negative instead? A valid folder should result in false

Reply:
Done.